assets/javascripts/discourse/routes/activity-pub-category.js

```diff
@@ -0,0 +1,51 @@
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import Category from "discourse/models/category";
+import DiscourseRoute from "discourse/routes/discourse";
+import ActivityPubActor from "../models/activity-pub-actor";
+
+export default DiscourseRoute.extend({
```

Comment:
Can we use native classes here, please? Since it's a new route, it's nice to do this now. Would look something like: 

```
export default class AccountCreatedIndex extends Route {
  @service: router;

  setupController(controller) {
  ...
  }
```

Reply:
Done https://github.com/angusmcleod/discourse-activity-pub/commit/6e35b7ba634cfbe1b43ba4afd966873752d66183